src/test/java/com/iexec/standalone/compute/pre/PreComputeServiceTests.java

```diff
@@ -419,5 +430,12 @@ void shouldAllTeeSessionGenerationErrorHaveMatch() {
                     .isNotNull();
         }
     }
+
+    @Test
+    void shouldFailedAndReturnNull() {
```

Comment:
Typo on method name here ?
```suggestion
    void shouldFailAndReturnNull() {
```